operate/qa/util/src/main/java/io/camunda/operate/qa/util/TestContainerUtil.java

```diff
@@ -99,6 +93,8 @@ public class TestContainerUtil {
   private static final Integer OPERATE_HTTP_PORT = 8080;
   private static final String DOCKER_ELASTICSEARCH_IMAGE_NAME =
       "docker.elastic.co/elasticsearch/elasticsearch";
+  private static final String ELS_DOCKER_TESTCONTAINER_URL_DEFAULT =
```

Comment:
I recommend keeping the comment that was associated with this property. It has been changed several times by other people and breaks the test containers each time, the comment is meant to explain why.